glean_parser/metrics.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         description: str,
         notification_emails: List[str],
         expires: Any,
+        defined_in: Optional[str] = None,
```

Comment:
*Very small nit*: Can this go at the end of the argument list (after `data_sensitivity`, but before `_config`) so this "output only" thing is separate from the ones we expect to find in the file?